src/cascadia/TerminalSettingsEditor/Actions.xaml

```diff
@@ -39,7 +41,7 @@ the MIT License. See LICENSE in the project root for license information. -->
                      to make sure it takes the entire width of the line -->
                 <ListViewItem HorizontalContentAlignment="Stretch"
                               AutomationProperties.Name="{x:Bind Name, Mode=OneWay}"
-                              AutomationProperties.AcceleratorKey="{x:Bind KeyChordText, Mode=OneWay}">
+                              AutomationProperties.AcceleratorKey="{x:Bind Keys, Mode=OneWay, Converter={StaticResource KeyChordToStringConverter}}">

```

Comment:
I originally wanted to make `KeyChord` `IStringable` and call it a day. But `KeyChord` is owned by `TerminalControl`, and I felt that it doesn't really make sense to add (de)serialization logic in there. If you disagree, let me know. The change would just require me to move `KeyChordSerialization` to `TerminalControl`.